src/utils/preparePrompt.ts

```diff
@@ -4,12 +4,19 @@ import * as vscode from "vscode";
 type PromptTypes = [{ title: string, type: string }];
 
 export const getPromptTypes = function (): PromptTypes {
+    // github:docker/labs-make-runbook?ref=main&path=prompts/docker
     const promptImage = vscode.workspace.getConfiguration('docker.make-runbook').get('prompt-image') as string;
     if (promptImage === "vonwig/prompts" || promptImage === "vonwig/prompts:latest") {
         spawnSync('docker', ['pull', "vonwig/prompts"]);
     }
-    const result = spawnSync('docker', ['run', '--rm', promptImage, "prompts"]);
-    return JSON.parse(result.stdout.toString());
+    const result = spawnSync('docker', ['run', promptImage, "prompts"]);
+    try {
+        return JSON.parse(result.stdout.toString());
+    }
+    catch (e) {
+        throw new Error(`Expected JSON from ${promptImage}, got STDOUT: ${result.stdout.toString()} STDERR: ${result.stderr.toString()} ERR: ${(result.error || "N/A").toString()}`);
```

Comment:
Similar to above.  Our prompts engine might fail but it should never write non application/json data to stdout.